notebooks/spark/AerospikeSparkPython.ipynb

```diff
@@ -28,11 +28,11 @@
    "outputs": [],
    "source": [
     "# IP Address or DNS name for one host in your Aerospike cluster\n",
-    "AS_HOST =\"172.16.39.141\"\n",
+    "AS_HOST =\"172.16.39.169\"\n",
     "# Name of one of your namespaces. Type 'show namespaces' at the aql prompt if you are not sure\n",
     "AS_NAMESPACE = \"test\" \n",
     "AS_FEATURE_KEY_PATH = \"/etc/aerospike/features.conf\"\n",
```

Comment:
Comment out the feature key line and add a comment that for connector versions 2.8 and 3.1 and later you do not have to provide the feature key.